install/roles/dovecot/tasks/main.yml

```diff
@@ -38,6 +38,16 @@
   vars:
     cert_dir: 'imap.{{ network.domain }}'
     entity_group: dovecot
+    access_private_key: true
```

Comment:
This is not needed.
In fact, it runs ok without any specific rights for the `dovecot` user.
The certificate and private key are loaded as root:
https://wiki.dovecot.org/SSL/DovecotConfiguration

Reply:
Thanks a lot for the reading, I will fix this today.

Reply:
OK, this is safer, I will remove it and run the tests again.

Reply:
OK, absolutely right, I remove the code, and everything works like a charm...